recipes/pybind11/all/test_package/conanfile.py

```diff
@@ -25,6 +23,10 @@ def generate(self):
         toolchain.variables["PYTHON_EXECUTABLE"] = PurePath(self._python_interpreter).as_posix()
         toolchain.generate()
 
+        env = Environment()
+        env.append_path("PYTHONPATH", os.path.join(self.build_folder, self.cpp.build.libdirs[0]))
+        env.vars(self, scope="run").save_script("testrun")
```

Comment:
Is testrun a standard name?

Reply:
Nope, in this case it's rather arbitrary. The `scope` here is what determines when it's called (during the runenv), but the name here is the filename it has. This is required for the test to run so I gave it a name to reflect that, but it can be butterflies.